tests/integration/docusaurus/expectations/creating_custom_expectations/expect_multicolumn_values_to_be_multiples_of_three.py

```diff
@@ -121,15 +121,12 @@ def validate_configuration(
         self, configuration: Optional[ExpectationConfiguration] = None
     ) -> None:
         super().validate_configuration(configuration)
-        if configuration is None:
-            configuration = self.configuration
-
-            try:
-                assert (
-                    "column_list" in configuration.kwargs
-                ), "column_list must be provided"
-            except AssertionError as e:
-                raise InvalidExpectationConfigurationError(str(e))
+        configuration = configuration or self.configuration
+
+        try:
+            assert "column_list" in configuration.kwargs, "column_list must be provided"
+        except AssertionError as e:
+            raise InvalidExpectationConfigurationError(str(e))
```

Comment:
Instead of an assert`, can't we just do:
```
if "column_list" not in configuration.kwargs:
    raise InvalidExpectationConfigurationError("column_list_must_be_provided")
```

Reply:
I would prefer to punt on this since this not really in the scope of this PR and is repeated many places. Also, once you change it in those places, you have to update all of the line numbers in the docs again!